superset-frontend/src/theme/ThemeController.ts

```diff
@@ -26,7 +26,7 @@ import {
   ThemeMode,
   themeObject as supersetThemeObject,
 } from '@superset-ui/core';
-import { normalizeThemeConfig } from '@superset-ui/core/theme/utils';
+import { normalizeThemeConfig } from '@apache-superset/core/ui/theme';
```

Comment:
It seems this function could be exported from `@apache-superset/core/ui` as it seems generic enough.